WordPress/Classes/ViewRelated/NUX/NUXSubmitButton.swift

```diff
@@ -0,0 +1,120 @@
+import UIKit
+import WordPressShared
+
+let NUXSubmitButtonDisabledAlpha = CGFloat(0.25)
+
+/// A stylized button used by NUX controllers. The button presents white text 
+/// surrounded by a white border.  It also can display a `UIActivityIndicatorView`.
+///
+@objc class NUXSubmitButton : UIButton
```

Comment:
That's odd, I would have expected them to overlap more. We can leave it for now as the purchase is still a work in progress.